### ISSUE TITLE
configure the + button on the top bar

### ISSUE THREAD

**Issue by adishourya** [PapyElGringo/material-awesome#27]
I am sure there's an easy way to do this or i must have missed something,but how can i configure the + sign on the top bar to launch rofi.

**Comment by BoredAbhi** [comment]
rofi can be launched by (Windows key + E) by default. 

To configure the "+" buttons you can go to  `.config/awesome/conf/tags.lua` in your home directory and edit the `defaultApp` value of the desktop of your choice to `rofi` or something similar.

**Comment by PapyElGringo** [comment]
Exact ! 
I have a plan for this button though ! Today the background of every tags are not used. I've plan to do an icon grid launcher with all the recent launched app for the current tag. And when you will press on the + button it's will minimized all window to make the grid appear !

What do you guys think about that?

**Comment by adishourya** [comment]
yeah. that would be cool .

**Comment by najamsk** [comment]
that would be like unity like experience dont you think? me personally happy with one defualt app. rest of the apps i luanch with roifi or keybindings.

**Comment by najamsk** [comment]
i think you should divert your attention to system tray like thing. like opening wifi manager on your icon, date time with picker. so they become more responsive and useful.

**Comment by PapyElGringo** [comment]
The current system require some configuration to be able to work so new users are forced to change the configs that's why I want to upgrade this system to make it's more usuable without any changes. It's will be different than unity because it's will display only tag related apps so it's won't be a huge grid with all the applications.

I'll dig in the widgets soon but I have some limitation for creating forms I don't have inputs or select yet.

**Comment by najamsk** [comment]
no worries about widgets people can always go back to nm-applet as you hinted. 

i got that recently used apps concept first time. it wont be huge grid true but still unity like.

People who want this beautiful setup you created I think wont mind learning about it. On the other hand it will make your life hell people asking same question again and again. might push you towards suicide or sainthood. 

thats why more documentation, FAQ's and videos so people can get up and running faster. 

having said all that its your project. make changes and commit the way you like. I am merely student learning from this project.

thanks.

**Comment by PapyElGringo** [comment]
Yeah documentation is my first focus atm Im gonna rewrite a bit the fiel architecture to make the documentation more easy !
I want this project to be community oriented so all feedback, suggestion and PR are very welcome !

Thanks for using this and building it with me
